OBC-Firmware-Onboarding.env

```diff
@@ -0,0 +1 @@
+CC_FOLDER_ROOT=C:/Users/natha/GNU-ARM-Embedded-Toolchain/arm-gnu-toolchain-11.3.rel1-mingw-w64-i686-arm-none-eabi
```

Comment:
This file should be called ".env", not "OBC-Firmware-Onboarding.env".

The reason for this is that the `Makefile` is dependent on `.env`